lib/gateways/db.js

```diff
@@ -17,6 +18,7 @@ export default (() => {
           options.password = process.env.PASSWORD;
           options.database = process.env.DATABASE;
         }
+        */
```

Comment:
I think we need to remove this before merging. Just noting so we remember.

Reply:
Good spot, and yes, that is correct. The staging/prod. environment relies on these environment variables which are defined in the Lambda functions.